### PR TITLE
[IMP] hosting: add admonition block on subscription

### PR DESCRIPTION
**Pull request by tiku-odoo**
Tasks:

https://www.odoo.com/web#id=3784176&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

https://www.odoo.com/web#id=3791421&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

Per change from FHE

### CODE REVIEW

File: content/administration/hosting.rst [hunk 1]
```diff
@@ -59,6 +59,11 @@ Transferring an Odoo Online database
       Click the gear icon (:guilabel:`⚙`) next to the database name on the `Odoo Online database
       manager <https://www.odoo.com/my/databases/>`_ to display its version number.
 
+   .. warning::
+      If there is an active Odoo subscription linked to the database being migrated reach out to the
```
Comment (xpl-odoo) [inline content/administration/hosting.rst:63]:
```suggestion
      If there is an active Odoo subscription linked to the database being migrated, reach out to the
```

**Comment by tiku-odoo** [comment]
@robodoo r+

**Comment by tiku-odoo** [comment]
@robodoo retry

**Comment by tiku-odoo** [comment]
@robodoo r+

**Pull request closed without merging**